### PR TITLE
(MAINT) added preserved hosts doc

### PR DESCRIPTION
**Pull request by kevpl**
[skip ci]

### CODE REVIEW

File: docs/how_to/preserve_hosts.md [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+## How Do I Use It?
+
+Note that where you decide to use this option will affect its precedence. You
+can learn more about this in our
+[argument processing & precedence doc](../topics/argument_processing_and_precedence.md).
+
+### Command-Line Option
+
+The primary way that this option is provided is through the command line, where
+it's represented by the `--preserved-hosts` options. This option defaults to
```
Comment (tvpartytonight) [inline docs/how_to/preserve_hosts.md]: nitpick: `options` should be `option`.

File: docs/how_to/preserve_hosts.md [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+
+## What Does It Do (In Detail)?
+
+Using the preserved hosts functionality outputs a `hosts_preserved.yml` file into
+the directory that holds all of the log files for a particular beaker run. The
+short way to access that directory is to look into the `log/latest` path, which
+is a symlink to the actual log directory for the run.
+
+The differences between the run that generates a preserved hosts file and later
+runs are listed below:
```
Comment (tvpartytonight) [inline docs/how_to/preserve_hosts.md]: I'm a little bit confused here about the concept of a run here, and think you just mean the file instead of run. Maybe rephrase to say something more like "A preserved_hosts.yml changes these options from the initial run". I say this because a subsequent "run" could still be modified by any additional flags or ENV VARs set.

File: docs/how_to/preserve_hosts.md [hunk 3]
```diff
@@ -0,0 +1,81 @@
...
+    /.../beaker --hosts log/centos7-64mdac-ubuntu1504-64a/2016-07-18_13_42_44/hosts_preserved.yml --tests smoke_simple.rb --preserve-hosts onpass
+
+The important piece here is the hosts argument, in that it's informing you of
+where the preserved hosts file is located. Of course, until you kick off another
+beaker run, you can also reach this file with the `log/latest/hosts_preserved.yml`
+symlink, as mentioned at the beginning of this section.
+
+## What Do I Do With It?
+
+For future runs, if you want to set something, you'll have to pay attention to
```
Comment (tvpartytonight) [inline docs/how_to/preserve_hosts.md]: Nitpick: I like the term `subsequent` here instead of `future`.

**Comment by kevpl** [comment]
@tvpartytonight look good?

**Comment by tvpartytonight** [comment]
👍
